client/coniksclient/internal/cmd/run.go

```diff
@@ -0,0 +1,198 @@
+package cmd
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+const help = "\n- register [name] [key]:\r\n" +
```

Comment:
> Why does it only start off w/ \n whereas the rest are \r\n?

Don't know but it seems like the carriage-return wasn't neccessary here (I tested it, it worked). I though an extra newline looked nicer. But it's also fine now. Thanks @c633